src/shared/types/props/epic.ts

```diff
@@ -94,6 +94,7 @@ export const variantSchema = z.object({
     choiceCardsSettings: choiceCardsSettings.nullish(),
     choiceCardAmounts: selectedAmountsVariantSchema.optional(), // deprecated, use choiceCardsSettings
     defaultChoiceCardFrequency: contributionFrequencySchema.optional(), // deprecated, use choiceCardsSettings
+    isCollapsible: z.boolean().optional(),
```

Comment:
we actually need this field on the Banner model - it's to replace this logic: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx#L167